src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -287,7 +287,7 @@ internal sealed class MemberSignatureComparer : IEqualityComparer<Symbol>
             considerReturnType: true,
             considerTypeConstraints: false,
             considerCallingConvention: true,
-            refKindCompareMode: RefKindCompareMode.ConsiderDifferences | RefKindCompareMode.AllowRefReadonlyVsInMismatch, // PROTOTYPE: Figure out how to test this.
+            refKindCompareMode: RefKindCompareMode.ConsiderDifferences | RefKindCompareMode.AllowRefReadonlyVsInMismatch,
```

Comment:
Removed the prototype comment here - seems it's not possible to test this. (It probably doesn't even matter whether the "ref readonly mismatch" bit is set.) Because if this matcher doesn't succeed, a fallback one which doesn't consider differences is used instead - the only difference is whether [an exact match is found](https://github.com/dotnet/roslyn/blob/e85f97f5d02a0840effd56c685f8a5ea5e68d5bd/src/Compilers/CSharp/Portable/Symbols/OverriddenOrHiddenMembersHelpers.cs#L611) but that doesn't matter anyway when the member is not from compilation in which case this matcher isn't even used.